Makefile

```diff
@@ -18,8 +17,8 @@ stable-mir-json:
 
 # generate smir and parse given test files (from parameter or run-rs subdirectory)
 smir-parse-tests: TESTS = $(shell find $(PWD)/kmir/src/tests/integration/data/run-rs -type f -name "*.rs")
-smir-parse-tests: SMIR = cargo -Z unstable-options -C deps/stable-mir-json run -- 
-smir-parse-tests:
+smir-parse-tests: SMIR = cargo -q -Z unstable-options -C deps/stable-mir-json run -- 
```

Comment:
Added a `-q` here so we don't get cluttered output per test.
(when debugging problems we can ovveride the variable)